feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SessionDetailFragment.kt

```diff
@@ -91,12 +91,9 @@ class SessionDetailFragment : DaggerFragment() {
                         )
                     )
                 }
-                R.id.session_place ->
-                    Toast.makeText(
-                        requireContext(),
-                        "not implemented yet",
-                        Toast.LENGTH_SHORT
-                    ).show()
+                R.id.session_place -> {
+                    this.findNavController().navigate(R.id.action_session_detail_to_floormap)
```

Comment:
Can you use a field of navController? 🙏 
https://github.com/DroidKaigi/conference-app-2019/blob/e043945a0ea696dd180c9c81b7ab706a1f4e2976/feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SessionDetailFragment.kt#L53

Reply:
I think you can use NavDirections to navigate? Can't you? 👀 
SessionDetailFragmentDirections

Reply:
Thanks!
I was careless though I noticed the existence of field navController. fixed it. 🙇 
And rewritten to use `SessionDetailFragmentDirections#actionSessionDetailToFloormap`